CHANGELOG.md

```diff
@@ -7,8 +7,9 @@
 - Add new openScope emblem vector graphic [#572](https://github.com/openscope/openscope/issues/572)
 - Adds additional meta tags to index.html head [#484](https://github.com/openscope/openscope/issues/572)
 - Adds a link to the full command reference at the end of the tutorial [#581](https://github.com/openscope/openscope/issues/581)
+- The mouse button to drag the radar screen is now right click [#564](https://github.com/openscope/openscope/issues/564)
 - The distance separator behind aircraft on ILS is now toggleable [#536](https://github.com/openscope/openscope/issues/536)
-- Removes the blue line "departure window" [#411](https://github.com/openscope/openscope/issues/411)
+- Removes the blue line "departure window" [#411](https://github.com/openscope/openscope/issues/411
```

Comment:
we need that closing `)` here to close the markdown link

Reply:
don't know how that happened!

fixed